lib/github/ldap.rb

```diff
@@ -65,6 +65,7 @@ class Ldap
     #   which to attempt opening connections (default [[host, port]]). Overrides
     #   host and port if set.
     # encryption: optional string. `ssl` or `tls`. nil by default
+    # tls_options: optional hash with TLS options for encrypted connections. Empty by default.
```

Comment:
It's good the tests show an example of this, but could you reference the ruby OpenSSL module here? Don't have to link to a specific version, but would help with googling later.

For reference, peer verification is described in http://ruby-doc.org/stdlib-2.0.0/libdoc/openssl/rdoc/OpenSSL.html#module-OpenSSL-label-Peer+Verification